### PR TITLE
Adds haxe 3.3-rc1 to AppMan

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: appman.xml [hunk 1]
```diff
@@ -1,5 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Entries xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Entry>
+		<Id>haxe3setup</Id>
+		<Name>Haxe + Neko</Name>
+		<Version>3.3.0</Version>
```
Comment (Gama11) [inline appman.xml]: Shouldn't this be `3.3.0-rc.1`? 3.3.0 is gonna be a separate release.

Comment (SlavaRa) [inline appman.xml]: fixed

**Comment by elsassph** [comment]
I'd like to wait for the official 3.3.0 and not add the RC(s)

**Comment by SlavaRa** [comment]
okay

**Pull request closed without merging**